gensrc/thrift/PlanNodes.thrift

```diff
@@ -465,11 +465,17 @@ struct TFrontendsMetadataParams {
   1: optional string cluster_name
 }
 
+struct TQueriesMetadataParams {
+  1: optional string cluster_name
+  2: optional bool   relayToOtherFE
```

Comment:
according to the code ruls, please rename relayToOtherFE to relay_to_other_fe

Reply:
fixed